crowbar_framework/config/puma.rb

```diff
@@ -31,6 +31,17 @@
 
 bind "tcp://#{LISTEN}:#{PORT}"
 
+before_fork do
+  PumaWorkerKiller.config do |config|
+    config.ram           = `free -m`.split(" ")[7]
```

Comment:
you can remove the `" "`, but IMHO reading `/proc/meminfo` and using a regex to get the `MemTotal` line is a more robust solution